### ISSUE TITLE
Item Settings

### ISSUE THREAD

**Issue by neruve** [finndev/PokeBuddy#28]
Is there a way we can change the item counts for recycled items. Like I would like to set my own limits for what it keeps (basically I wanna keep all the ultraballs I can).

I've noticed the items settings are blank. I've also noticed the pokemon settings don't actually work.

Nonetheless, amazing work from the developer. It is greatly appreciated.

**Comment by howcanihelp** [comment]
I agree with neruve, amazing work!

I also have a problem with the item settings, I can't see anything at all. Just downloaded the latest version.
![buddy](https://cloud.githubusercontent.com/assets/20733209/17266786/075be312-55d3-11e6-89c0-11bccf864155.png)


Not related to items, is it possible to automatically save the last position before closing the software?
I just got soft banned because I closed and then reopened the software and start the bot without noticing it got back to the initial GPS settings.
